src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs

```diff
@@ -87,6 +88,8 @@ protected override Task CleanupClientConnections(string fromInstanceId = null)
                     continue;
                 }
 
+                ClientInvocationManager.Router.CleanupInvocationsByConnection(connection.Key);
```

Comment:
Add net7.0 limit? Otherwise it'll throw.

Reply:
done